test/packages/good_v3/data_stream/foo/fields/some_fields.yml

```diff
@@ -89,3 +89,8 @@
 - name: empty_expected_values
   type: keyword
   expected_values: []
+- name: some_counted_keyword
+  type: counted_keyword
```

Comment:
It looks like we won't need changes in Fleet, I have tried this definition, and Fleet generates the mapping below, what seems enough. One thing less to do :slightly_smiling_face: 
```
- name: event.counted
  type: counted_keyword
```
```
              "counted": {
                "type": "counted_keyword"
              },
```

Reply:
So you mean that the second part you were referring to in [this comment](https://github.com/elastic/package-spec/issues/698#issuecomment-1906367674) is not necessary?

> Also, the change will need to be included in a new package spec minor, probably 3.1, and this version should be used in the kibana configuration for serverless deployments that support it, this can be updated in these files: https://github.com/elastic/kibana/blob/ee11d8bbce96acfc4f1eeff5588087c360b74ab9/config/serverless.oblt.yml#L44

Reply:
Well, I would still keep it in 3.1 because this is a new feature in the stack, even if it doesn't seem to need explicit support in Kibana.